app/views/candidates/home/index.html.erb

```diff
@@ -87,6 +87,11 @@ Visit schools to see how teachers plan lessons, manage classrooms and teach subj
         </h2>
 
         <ul class="govuk-list govuk-!-font-size-16 govuk-se-spaced-list">
+          <li>
+            <a href="<%= candidates_what_to_expect_path %>" class="govuk-!-font-weight-bold">
```

Comment:
Same here, could we change the path to school_experience_guide_for_candidates ? @leoapost